amplify/backend/function/getUserReputation/src/index.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const { getColonyNetworkClient, Network } = require('@colony/colony-js');
+const { providers } = require('ethers');
+
+const ROOT_DOMAIN_ID = 1; // this used to be exported from @colony/colony-js but isn't anymore
+const RPC_URL = 'http://network-contracts.docker:8545'; // this needs to be extended to all supported networks
+
+/**
+ * @type {import('@types/aws-lambda').APIGatewayProxyHandler}
+ */
+exports.handler = async (event) => {
```

Comment:
Please set `ethers`'s log level to `ERROR` only, to remove the "Duplicate contract definition" warnings in the console:
![Screenshot from 2022-11-07 13-43-14](https://user-images.githubusercontent.com/1193222/200302530-2e153000-4792-4b8d-ba1f-0518a69b4dad.png)

You do this by importing the `Logger` class from `ethers` and passing it a error config level:
```js
import { utils } from 'ethers/utils';

const { Logger } = utils;

Logger.setLogLevel(Logger.levels.ERROR);
```